tests/expectations/metrics/test_core.py

```diff
@@ -333,6 +338,66 @@ def test_column_quoted_name_type_sa(sa):
         )
 
 
+def test_column_quoted_name_type_sa_handles_explicit_string_identifiers(sa):
+    """
+    Within SQLite, identifiers can be quoted using one of the following mechanisms:
+    'keyword'		A keyword in single quotes is a string literal.
+    "keyword"		A keyword in double-quotes is an identifier.
```

Comment:
@TrangPham : If this is for SQL, then perhaps we need to reproduce how a user with a command-line SQL editor would handle the column names with the spaces in them.  Could we find a definitive recommendation from, say, ANSI SQL documentation?  Thank you.

Reply:
Unfortunately I haven't seen that the ANSI is used in most places.

Reply:
@TrangPham Then I wonder if this should be a reason for us to disallow columns/tables with spaces in their names?  If we do not have anything definitive for ANSI SQL, is there anything for Snowflake?  MS SQL Server?  Oracle? Etc. -- the dialects, like you said.  Also, does Spark API mention these cases?  It seems very painful to handle!

Reply:
As discussed in slack - we should handle column names that require delimited identifiers. I updated the code to specifically handle this for SparkDF.